src/fragments/lib/in-app-messaging/js/getting-started.mdx

```diff
@@ -0,0 +1,125 @@
+Make sure you have completed the below steps:
+
+- [Completed the pre-requisite steps.](/lib/in-app-messaging/prerequisites)
+- [Create a campaign from the pinpoint console.](/lib/in-app-messaging/create-campaigns)
+
+## Integrate your application
+
+In your application directory, you should first install the necessary dependencies. Although Amplify In-App Messaging can be used as a standalone JavaScript library, the only UI integration currently available is only for _React Native_. The integrations steps here assume a React Native application as the implementation target.
```

Comment:
Although Amplify In-App Messaging can be used as a standalone JavaScript library, the ~only~ UI integration **is** currently available ~is~ only for _React Native_.